### PR TITLE
Make path.sh to dynamically use the NDK version specified in depin…

### PR DESCRIPTION
**Pull request by gakramx**
Remove hardcoded NDK version and replaced it with a reference to the version defined in depinfo.sh

### CODE REVIEW

File: buildscripts/include/path.sh [hunk 1]
```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 
+# Get the directory of the current script
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+. "$SCRIPT_DIR/depinfo.sh"
```
Comment (sfan5) [inline buildscripts/include/path.sh:6]: this is a bit redundant, you can just use `$DIR/include/depinfo.sh`

Comment (gakramx) [reply buildscripts/include/path.sh:6]: Thanks for the review.
I opted for `$SCRIPT_DIR/depinfo.sh` over `$DIR/include/depinfo.sh `to ensure the script's portability and reliability.

 Using `$SCRIPT_DIR` guarantees that the script works regardless of its location, enhancing readability by explicitly stating the dependency's location relative to the script. 

Would you like me to continue with this approach, or do you have a preference for using `$DIR`?

Comment (sfan5) [reply buildscripts/include/path.sh:6]: It's not like this script is designed to be moved, so yes I prefer relying on `$DIR`.

Comment (gakramx) [reply buildscripts/include/path.sh:6]: Thank you i get it , I will update it now.

**Comment by sfan5** [comment]
Looks good, can you squash the commits?

**Comment by gakramx** [comment]
> Looks good, can you squash the commits?

Sorry i forget this , I squashed 3 commits into one !

**Comment by syphyr** [comment]
I'm not sure why $DIR is needed. All the other scripts just use `. ./include/depinfo.sh`

**Comment by sfan5** [comment]
merged as 1b8d2935229f607c12e7b26863a741bbec6d5a99

**Comment by gakramx** [comment]
> I'm not sure why $DIR is needed. All the other scripts just use `. ./include/depinfo.sh`

Using `$DIR` ensures consistent and reliable sourcing of **depinfo.sh**, regardless of the script's location.

**Pull request closed without merging**